aten/src/ATen/native/cpu/radix_sort.h

```diff
@@ -0,0 +1,196 @@
+#pragma once
+#include <ATen/Config.h>
+
+#if !AT_PARALLEL_OPENMP
+
+namespace at { namespace native {
```

Comment:
We are in C++17 world now, so using nested namespaces is fine
```suggestion
namespace at::native {
```